js/src/struct.js

```diff
@@ -43,7 +43,12 @@ export const fieldNameRe = new RegExp(fieldNameComponentRe.source + '$');
  *   setS(value: string): MyStruct;
  * }
  *
- * To reflect over structs you can create a new StructMirror.
+ * With one major exception and that is that if the field name conflicts with any of the properties
```

Comment:
nit: this would read slightly better as "With one major exception: if the field name conflicts..." rather than "With one major exception and that is that if the field name conflicts..."